src/Components/AffectedClustersTable/AffectedClustersTable.spec.ct.js

```diff
@@ -203,7 +203,18 @@ describe('non-empty successful affected clusters table', () => {
     });
   });
 
+  it('sorting the last seen column', () => {
+    cy.get(TABLE)
+      .find('td[data-key=2]')
+      .children()
+      .eq(0)
+      .should('have.text', '2 years ago');
```

Comment:
I have this example of tests in which after sorting, it compares them by description: https://github.com/RedHatInsights/ocp-advisor-frontend/pull/91/files#diff-15915e05fb299bab67d48b0c638eeee4d7347855f7eb4da1688fedc2e2257fd8R165

Beware that for those tests, what I did was to sort first by description and later by the other column, so that ordering is consistent

Reply:
Yep you are correct, changed the tests so it was "time independent"